src/node.cc

```diff
@@ -2137,6 +2137,32 @@ void Hrtime(const FunctionCallbackInfo<Value>& args) {
   fields[2] = t % NANOS_PER_SEC;
 }
 
+// CPUUsage exposes libuv's uv_getrusage() this-process resource usage accessor,
+// to access ru_utime (user CPU time used) and ru_stime (system CPU time used),
+// which are uv_timeval_t structs (long tv_sec, long tv_usec) as tuples
+// similar to process.hrtime() values.  Only it's seconds/microseconds instead
+// of hrtime()'s seconds/nanoseconds.
```

Comment:
To avoid confusion in user land, would it make sense to turn them into nanoseconds?